auth/utils/helpers.ts

```diff
@@ -122,25 +108,49 @@ export function removeSuffix(str: string, suffix: string): string {
 }
 
 /**
- * Checks for at least one of two elements being defined.
+ * Checks that exactly one of the arguments provided is defined.
+ * Returns true if one argument is defined. Returns false if no
+ * argument are defined or if 2 or more are defined.
  *
- * @param a - the first object
- * @param b - the second object
- * @returns true if a or b is defined; false if both are undefined
+ * @param args - The spread of arguments to check
+ * @returns true if and only if exactly one argument is defined
  */
-export function atLeastOne(a: any, b: any): boolean {
-  return Boolean(a || b);
+export function onlyOne(...args: any): boolean {
+  return countDefinedArgs(args) === 1;
```

Comment:
I changed all of these functions to accept spread arguments instead of binary options.

The check needed for this authenticator (exactly one out of three) prompted me to think these functions would be more valuable down the road if they weren't limited to looking at only two options. So, now they can take any number of options and still perform the same checks. They are perfectly compatible with their old implementations, though they've technically "changed" in the API.